Text/Internal/Css.hs

```diff
@@ -345,24 +353,28 @@ blockToMixin :: Name
              -> Block Unresolved
              -> Q Exp
 blockToMixin r scope (Block _sel props subblocks mixins) =
+    -- TODO: preserve the CPS in @mixinBlocks@ below
     [|Mixin
         { mixinAttrs    = concat
                         $ $(listE $ map go props)
                         : map mixinAttrs $mixinsE
-        -- FIXME too many complications to implement sublocks for now...
-        , mixinBlocks   = [] -- foldr (.) id $(listE $ map subGo subblocks) []
-        }|]
-      {-
-      . foldr (.) id $(listE $ map subGo subblocks)
-      . (concatMap mixinBlocks $mixinsE ++)
+        , mixinBlocks   = foldr (++) [] $(listE $ map subGo subblocks)
```

Comment:
Is there a difference between `foldr (++) []` and `concat`?

Reply:
Not that I can think of. I think I was stuck trying to rewrite this as a fold.